### DISCUSSION TITLE
Golden Bonsai Plant

### DISCUSSION THREAD

**Discussion by Lvegas01** [sunflower-land/sunflower-land#2374] [Ideas]
The proposal consists of planting the Golden Bonsai directly on the ground. Generating a benefit of +0.25 Gold every 24h and in the future add +1 fertilizer (I don't know which one) and allow the benefit to increase to +0.5 each time it is applied. A maximum of 1 or 2 golden bonsai are allowed in each land.
Thank you.

**Comment by brian-man** [comment]
Adding boosts to existing decorative collectibles is at best foolish IMO.

For the specific example of Golden Bonsai, there are people that have collected dozens of them.

**Comment by Lvegas01** [comment]
Of course, that is why I limit that only 1 or a maximum of 2 are allowed to be planted, whoever wants to have 100 ornaments can have them

**Reply by brian-man** [reply]
> Of course, that is why I limit that only 1 or a maximum of 2 are allowed to be planted, whoever wants to have 100 ornaments can have them

That doesn't change the rest of what I said.  I have a number of decorations (including Golden Bonsai) and I don't want any of them to "after the fact" get boosts.  It's bad for the game in a number of ways.

**Comment by offerT** [comment]
cows can produce fertilizer once a day, and producing milk every three days is a good idea

**Comment by TheDcol** [comment]
Golden bonsai is a decorative item with no intention to add boosts to it.

There are already plenty enough ways to get free stuff in the game.. adding free gold everyday would be too much

**Comment by ghost** [comment]
Explore my idea for the decorations: https://github.com/sunflower-land/sunflower-land/discussions/2393 
Such a system is easier to balance, and it can give minor boosts, which will not greatly imbalance the game.
